pkgs/development/python-modules/block-io/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchPypi, buildPythonPackage, base58, ecdsa, pycrypto, requests, six }:
+
+buildPythonPackage rec {
+  name = "${pname}-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.